### PR TITLE
added sweet fade-in and implemented react-router, deleted fetch helper

### PR DESCRIPTION
**Pull request by tylerjhevia**
Deleted the fetch helper file that we're no longer using, added react-router and NavLinks to the login and movie pages, and added that sweet sweet fade-in

### CODE REVIEW

**Comment by jackmallahan** [comment]
sweet action

File: app/components/MovieCard/MovieCard.css [hunk 1]
```diff
@@ -1,20 +1,55 @@
...
+  margin: 20px;
+  height: 250px;
+  width: 500px;
+  opacity: .5;
+  background-size: cover;
+  background-repeat: no-repeat;
+  animation: fadeIn ease-in 1;
+  animation-duration: .75s;
+  animation-fill-mode: forwards;
+  -moz-animation: fadeIn ease-in 1;
```
Comment (jackmallahan) [inline app/components/MovieCard/MovieCard.css:15]: Do we need all of this webkit stuff? Who uses firefox, anyway?

File: app/components/MovieCard/MovieCard.css [hunk 2]
```diff
@@ -1,20 +1,55 @@
...
+@-moz-keyframes fadeIn {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+@keyframes fadeIn {
```
Comment (jackmallahan) [inline app/components/MovieCard/MovieCard.css:40]: That's a sweet fade.

**Comment by jbexx** [comment]
killer fade in tyler!  why doesn't the opacity work anymore?  hmm?

File: app/components/MovieCard/MovieCard.css (hunk 1 above)
Comment (tylerjhevia) [reply app/components/MovieCard/MovieCard.css:15]: I don't know, what do you think @jbexx ?

File: app/components/App/App.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
+header {
+  background-color: orange;
```
Comment (jbexx) [inline app/components/App/App.css:2]: oh man,  I love orange.  Go Tigers.  (gators suck)

**Comment by jackmallahan** [comment]
@tylerjhevia haters gon hate. Don't listen to @jbexx

File: app/components/App/App.css (hunk 3 above)
Comment (jackmallahan) [reply app/components/App/App.css:2]: You can't pick and choose teams. COLORS DONT FADE

File: app/components/MovieCard/MovieCard.css (hunk 1 above)
Comment (jbexx) [reply app/components/MovieCard/MovieCard.css:15]: I think my grandmother uses firefox.  @jackmallahan, do you not care about my grandmother?

File: app/components/MovieCard/MovieCard.css (hunk 2 above)
Comment (jbexx) [reply app/components/MovieCard/MovieCard.css:40]: its pretty sweet, but not that sweet

**Pull request merged**